protobufs/livekit_sip.proto

```diff
@@ -320,29 +358,39 @@ message SIPDispatchRule {
 }
 
 message CreateSIPDispatchRuleRequest {
-  SIPDispatchRule rule = 1;
+  SIPDispatchRuleInfo dispatch_rule = 8; // Rule ID is ignored
+
+  SIPDispatchRule rule = 1 [deprecated=true];
```

Comment:
hmm I think this will make the API more complicated.. nested seems harder to use to me.

Reply:
yeah, I'm talking about the nesting. I think CLI could hide it, just thinking through for SDK integrations. while this is more consistent, the SDK update cycle is a bit painful. Your call though on the path forward.